README.md

```diff
@@ -28,7 +17,7 @@ The `main()` function is the single thread in the application. It toggles the st
 ## Building and running
 
 1. Connect a USB cable between the USB port on the board and the host computer.
-2. <a name="build_cmd"></a> Run the following command to build the example project and program the microcontroller flash memory:
+1. <a name="build_cmd"></a> Run the following command to build the example project and program the microcontroller flash memory:
```

Comment:
what is the use of the tag?

Reply:
Markdown is taking care of incrementing the index values. And in this readme itself at the beginning the used the tag with "1.",  so made as to the same format

Reply:
I was asking about <a name="build_cmd"></a>